src/OpenSheetMusicDisplay/OSMDOptions.ts

```diff
@@ -178,6 +178,11 @@ export interface IOSMDOptions {
      *  at different measures. So this option may result in a system break after a single measure in a system.
      */
     newSystemFromXML?: boolean;
+    /** Whether to begin a new system ("line break") when given a new page in XML ('new-page="yes"').
```

Comment:
when given a new page, but newPageFromXML is false.
Again, when we do a page break because the newPageFromXML is true, we don't need a system break. A page break is an implicit system break.